packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config-table.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+	<Datatable
+		:value="data"
+		v-model:expanded-rows="expandedRows"
+		dataKey="id"
+		:row-class="rowClass"
+		class="p-datatable-sm"
+		:class="{ 'hide-header': hideHeader }"
+	>
+		<Column expander style="width: 5rem" />
+		<Column field="id" header="ID" style="width: 10%"></Column>
+		<Column field="name" header="Name" style="width: 15%"></Column>
+		<Column field="type" header="Value type" style="width: 10%">
+			<template #body="slotProps">
+				<Button
+					text
+					v-if="slotProps.data.type === ParamType.MATRIX"
+					icon="pi pi-table"
+					@click="openMatrixModal(tableType, slotProps.data.id)"
+				/>
+				<span v-else-if="slotProps.data.type === ParamType.EXPRESSION">Expression</span>
+				<Dropdown
+					v-else
+					:model-value="slotProps.data.type"
+					:options="typeOptions"
+					optionLabel="label"
+					optionValue="value"
+					placeholder="Select a parameter type"
+					@update:model-value="(val) => changeType(slotProps.data.value, val)"
+				/>
+			</template>
+		</Column>
+		<Column field="value" header="Initial value" style="width: 15%">
```

Comment:
I would consider renaming this to just `Value` - "initial values" means they can change during the process of a simulation, but these are just constants or sampling distributions.